rest/taskana-rest-spring/src/main/java/pro/taskana/task/rest/TaskController.java

```diff
@@ -531,7 +532,8 @@ public ResponseEntity<TaskRepresentationModel> terminateTask(@PathVariable Strin
    * @title Transfer a Task to another Workbasket
    * @param taskId the Id of the Task which should be transferred
    * @param workbasketId the Id of the destination Workbasket
-   * @param setTransferFlag sets the tansfer flag of the task (default: true)
+   * @param transferTaskRepresentationModel sets the transfer flag of the task (default: true) and
```

Comment:
Please capitalize Task and Workbasket

Reply:
Did you mean Task and Owner? I will capitalize them

Reply:
I only meant Task. Owner isn't an entity and shouldn't be capitalized